numpy/_core/_add_newdocs.py

```diff
@@ -6192,6 +6192,9 @@
 
 add_newdoc('numpy._core.multiarray', 'dtype',
     """
+    dtype(dtype, align=False, copy=False, **kwargs)
```

Comment:
I tried using `metadata={}` before, but passing `metadata={}` isn't quite the same as omitting it.

It also looked a bit off in the docs; not sure why though.
<img width="738" height="228" alt="image" src="https://github.com/user-attachments/assets/128ff89e-1b41-4566-b707-ba17c94b6067" />

Reply:
> I tried using metadata={} before, but passing metadata={} isn't quite the same as omitting it.

Hmmm, that is silly, we could (and should) allow `metadata=None` and translate that to NULL.
But yeah, that is slightly independent from this and fine.